lib/FastANI/utils/fast_ani_report.py

```diff
@@ -20,7 +20,8 @@ def create_report(callback_url, scratch, workspace_name, result_data):
     os.mkdir(html_dir)
     # Move all pdfs into the html directory
     for result in result_data:
-        os.rename(result['viz_path'], os.path.join(html_dir, result['viz_filename']))
+        if os.path.exists(result['viz_path']):
```

Comment:
Found that in rare cases, fastani's Rscript for generating the visualization silently fails for no debuggable reason. So I just quietly continue.